### PR TITLE
Adding explicit SOAP dependency

### PR DESCRIPTION
**Pull request by BrekiTomasson**
This package throws exceptions if the PHP extension for SOAP is not installed.

### CODE REVIEW

**Comment by dannyvankooten** [comment]
Hello @BrekiTomasson,

Thank you for your PR! I merged it at first, then later realized that `ext-soap` is already a suggested (but not strictly required) package in [composer.json](https://github.com/ibericode/vat/blob/master/composer.json#L37). It's only required if validating VAT numbers by existence, but this library can definitely be used without it.

Perhaps we should add a check for the extension in the one method where this extension is actually required and then show a more helpful error message? Or can we somehow utilize Composer process to show a heads-up about this (maybe it already does this)?

**Comment by BrekiTomasson** [comment]
Your call, of course, although I would personally have kept `ext-soap` as a required package since the feature - while optional - breaks in a very unintuitive way if `ext-soap` is not installed.

If you want to keep it as a suggested package, I would most likely go for both of your suggestions above - both a suggestion in `composer` and a more helpful error message. Your current line in `composer.json` does the job for the first part, but I suggest rewording it to something more like "Required for VIES VAT number validation".
